easybuild/easyconfigs/p/polars/polars-1.27.1_polars-utils.patch

```diff
@@ -0,0 +1,24 @@
+Fix polars-utils crate.
```

Comment:
Where does fix come from (and what does it fix exactly, any relevant error messages we can mention)?

Any reference we can add here (PR, issue, etc.)?